scrapyd_k8s/api.py

```diff
@@ -148,5 +189,11 @@ def run():
     if config_username is not None and config_password is not None:
         enable_authentication(app, config_username, config_password)
 
+    pod_watcher_thread = threading.Thread(
+        target=watch_pods,
+        kwargs={'namespace': scrapyd_config.get('namespace', 'default')}
```

Comment:
I think there are now multiple places where we could get the namespace, and provide a default. I think I might add a `scrapyd_config` method to get the namespace, and provide the default there, so that this is consistent across the places where the namespace is used.

Reply:
Please, let me know if you changed it so I can adopt your edits, or if you want I can add it

Reply:
Note that for k8s there already is a namespace setting on the configuration. Sorry, I meant that you could add this method, if you think that's a good idea :) To avoid the default namespace being set in multiple places, please make a method in the configuration class, which also provides the default.